skodaconnect/const.py

```diff
@@ -50,7 +50,7 @@
     'X-Client-Id': XCLIENT_ID,
     'X-App-Version': XAPPVERSION,
     'X-App-Name': XAPPNAME,
-    'User-Agent': USER_AGENT,
+    #'User-Agent': USER_AGENT,
```

Comment:
I was still getting 429 until I  also commented out all other `User-Agent` occurrences in this file (5 or so).

Reply:
sidenote: I'm good with just commenting out the UA but FWIW it also works by setting it to what the MySkoda App is sending (= `MySkoda/230629002 CFNetwork/1474 Darwin/23.0.0`)